### PR TITLE
Rotate aes key. Re-encrypt files. Add npmignore.

### PR DESCRIPTION
**Pull request by kkortchinska**
**What does this PR do?** 
- Re-encrypts files with a new `aes` key. 
- Adds all files connected to docker migration to `.npmignore`

Based on [this](https://docs.npmjs.com/misc/developers#before-publishing-make-sure-your-package-installs-and-works) article. 

**Hint:** 
This command allows you to check what files are going to be published without actually publishing:
`npm pack && tar -xvzf *.tgz && rm -rf package *.tgz`

Result - all docker migration files are gone from the package: 
image

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+node_modules
+*.aes
+*.decrypted
+.ssh
+lib
+Dockerfile
+*.encrypted
+*.yml
```
Comment (simonhoye) [inline .npmignore:8]: Maybe we should be specific for the .yml files? There could be instances where we want to publish yml files?

Comment (kkortchinska) [reply .npmignore:8]: I guess in this repo it's pretty straightforward, but I am happy to update to be more specific :)

Comment (simonhoye) [reply .npmignore:8]: agree, I was just thinking of using this file across all the repos :)

**Comment by simonhoye** [comment]
Also you could remove the lib folder as its not used in this repo

**Comment by kkortchinska** [comment]
@simonhoye I've made the changes in the latest commit.

**Comment by kkortchinska** [comment]
> Nice one :)
> 
> Do we/should we have a docker ignore as well?

@camjc do we have examples of what we should be including in dockerignore?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,5 @@
 node_modules
 *.aes
 *.decrypted
-/.codeship/*/*
```
Comment (simonhoye) [inline .gitignore:4]: Sorry, but I think we should leave these as is... just to save having to add anything in the future... anything inside the .codeship folder should be ignored

**Comment by simonhoye** [comment]
Regarding @camjc's comment about the .dockerignore file... it's a good idea to add one, this is a good example https://github.com/ahmdigital/ahm-salesforce/blob/master/.dockerignore

**Comment by kkortchinska** [comment]
Thanks @simonhoye! 

- Added `.dockerignore` following example from `ahm-salesforce` repo 
- Left `.codeship` folder in `.gitignore` without change

**Pull request merged**